packages/e2e-tests/specs/editor/various/autosave.test.js

```diff
@@ -186,6 +186,49 @@ describe( 'autosave', () => {
 		toggleOfflineMode( false );
 	} );
 
+	it( 'should clear local autosave after successful save', async () => {
+		await clickBlockAppender();
+		await page.keyboard.type( 'before save' );
+		await saveDraft();
+
+		await page.keyboard.type( 'after save' );
+
+		// Trigger local autosave
+		expect( await page.evaluate( () => window.sessionStorage.length ) ).toBe( 0 );
+		await page.evaluate( () => window.wp.data.dispatch( 'core/editor' ).__experimentalLocalAutosave() );
+		expect( await page.evaluate( () => window.sessionStorage.length ) ).toBe( 1 );
+
+		// Save and wait for "Saved" to confirm save complete.
+		await Promise.all( [
+			page.waitForSelector( '.editor-post-saved-state.is-saved' ),
+			pressKeyWithModifier( 'primary', 'S' ),
+		] );
+
+		expect( await page.evaluate( () => window.sessionStorage.length ) ).toBe( 1 );
```

Comment:
Shouldn't the length be **0** in this case? The test description is:
> should **clear** local autosave after successful save

Reply:
Oh wow, how did that happen? 😰 I'll look. In the meantime, in actual usage the issue seemed to be fixed.